styles/responsive_styles.h

```diff
@@ -44,13 +46,15 @@ using ResponsiveDragL =
 // heating and intensity will increase or decrease based on turning hilt
 // TRANSITION1 & TRANSITION2 = transition Begin & End
 // SIZE1 & SIZE2 control MELT area limits for TwistAngle
+// LOCATION control SmoothStep location
 template<class COLOR = Mix<TwistAngle<>,OrangeRed,Red>,
          class TR1 = TrWipeIn<600>,
          class TR2 = TrWipe<600>,
          class SIZE1 = Int<4000>,
-         class SIZE2 = Int<10000>>
+         class SIZE2 = Int<10000>,
+         class LOCATION - Int<30000>>
```

Comment:
Did you test this?
Because it says - instead of = here and I don't think that will compile.

Reply:
My mistake, typo. I fixed in my branch does it show up or do I edit it here too?